tests/Kernel/fixtures/rendering.yml

```diff
@@ -1083,6 +1083,27 @@
       article.ecl-card div.ecl-card__body div.ecl-card__description: 0
     equals:
       article.ecl-card header.ecl-card__header h1.ecl-card__title .ecl-link: "Item title"
+- array:
```

Comment:
If we support length variable I would suggest to add one more test to check trimming of description field.

Reply:
I updated the existing one. Doesn't make any difference if there are two tests checking for description.